<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>麒麟AI助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="chatPage" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="ChatWindow" name="chatWindow" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rolePage" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="RoleManager" name="roleManager" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="actionSwitchToChat"/>
    <addaction name="actionSwitchToRole"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSwitchToChat">
   <property name="text">
    <string>聊天</string>
   </property>
  </action>
  <action name="actionSwitchToRole">
   <property name="text">
    <string>角色管理</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatWindow</class>
   <extends>QWidget</extends>
   <header>chatwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RoleManager</class>
   <extends>QWidget</extends>
   <header>rolemanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui> 